.github/workflows/ci.yml

```diff
@@ -126,7 +126,7 @@ jobs:
         run: pnpm vite build --mode=development
 
       - name: Set BrowserStack Local Identifier
-        if: startsWith(github.ref, 'refs/tags/v')
+        if: startsWith(github.ref, 'refs/tags/v') && endsWith(github.ref, '-ember-source')
```

Comment:
Apparently you still cannot comment on unchanged lines on github:

Do we want to run this workflow at all for changes and or tags of @glimmer/component?

Reply:
Yes. The CI workflow is good test coverage for everything together, including @glimmer/component. 

With #20751 we'll actually get better integration test coverage of @glimmer/component than we've ever had.

Reply:
Looking at this again, I think this is wrong -- the BrowserStack Local Identifier needs to be set on any tag, not just ember-source tags, if we're still running the BrowserStack jobs